app_dart/lib/src/service/scheduler.dart

```diff
@@ -508,7 +508,11 @@ class Scheduler {
 
   /// Checks if the framework only path is enabled
   static bool _isFrameworkOnlyAllowed(PullRequest pullRequest) {
-    // An allow-list or flag checking could be done here.
+    // TODO(matanlurey): Debug further why this doesn't work on release branches.
+    if (pullRequest.base?.ref case final branch? when branch != 'master') {
```

Comment:
This line is 100% unreadable.

if (Config.defaultBranch(commit.slug) != pullRequest.base?.ref) {
  // refused.
}

Reply:
K will fix